feat: FIT-761: New multiple annotators agreement ground truth endpoint

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/18605850279)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/648
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18542876039

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18564731978)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18564731978

mcanu committed:
Merge branch 'develop' into 'fb-fit-761-agreement-ground-truth'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18564731978

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18577518184)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18577518184

mcanu committed:
Merge branch 'develop' into 'fb-fit-761-agreement-ground-truth'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18577518184

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18594759395

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-fit-761-agreement-ground-truth'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18594759395

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18594984227

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18603474251

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-fit-761-agreement-ground-truth'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18603474251

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18603544722

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18603727535

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18605305608

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18605850279

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/18606555175
